scripts/sysman/firmware.yml

```diff
@@ -76,6 +76,7 @@ name: Flash
 details:
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
+    - "The application may call zesFirmwareGetFlashProgress() to get completion status."
```

Comment:
please expand that this is not a blocking call.

Reply:
Made this non blocking explicit. Let me know